test/tests.js

```diff
@@ -1072,6 +1072,53 @@ suite('Template Instantiation', function() {
     });
   });
 
+  test('Repeat Object Keys', function(done) {
+    var div = createTestHtml(
+      '<template repeat="{{ obj }}"">{{}}</template>');
+
+    var model = { obj: {"key1": 1, "key2": 2, "key3": 3} };
```

Comment:
thoughts on supporting ES6 Map? http://kangax.github.io/compat-table/es6/#Map
it's going to replace objects as the main way of doing maps (at least, that is what the EcmaScript designers are hoping :) )